content/resources/yum_repository/_index.md

```diff
@@ -21,135 +47,126 @@ resource_description_list:
-syntax_full_properties_list: null
-syntax_shortcode: null
-registry_key: false
-nameless_apt_update: false
-nameless_build_essential: false
-resource_package_options: false
+syntax_full_properties_list:
+- "`yum_repository` is the resource."
+- "`name` is the name given to the resource block."
+- "`action` identifies which steps Chef Infra Client will take to bring the node into
+  the desired state."
+- "``baseurl``, , ``clean_metadata``, ``cost``, ``description``, ``enabled``, ``enablegroups``,
+  ``exclude``, ``failovermethod``, ``fastestmirror_enabled``, ``gpgcheck``, ``gpgkey``,
+  ``http_caching``, ``include_config``, ``includepkgs``, ``keepalive``, ``make_cache``,
+  ``max_retries``, ``metadata_expire``, ``metalink``, ``mirror_expire``, ``mirrorlist``,
+  ``mirrorlist_expire``, ``mode``, ``options``, ``password``, ``priority``, ``proxy``,
+  ``proxy_password``, ``proxy_username``, ``repo_gpgcheck``, ``report_instanceid``,
+  ``repositoryid``, ``skip_if_unavailable``, ``source``, ``sslcacert``, ``sslclientcert``,
+  ``sslclientkey``, ``sslverify``, ``throttle``, ``timeout``, and ``username`` are
+  the properties available to this resource."
 actions_list:
   :nothing:
     shortcode: resources_common_actions_nothing.md
 properties_list:
 - property: baseurl
   ruby_type: String, Array
   required: false
-  default_value: null
-  new_in: null
+  default_value:
+  new_in:
   description_list:
-  - markdown: 'URL to the directory where the Yum repository''s ''repodata'' directory
-
-      lives. Can be an <http://>, <https://> or a <ftp://> URL. You can
-
-      specify multiple URLs in one baseurl statement.'
-- property: clean_headers
-  ruby_type: true, false
-  required: false
-  default_value: 'false'
-  new_in: null
-  description_list:
-  - markdown: '`DEPRECATED`
-
-
-      Specifies whether you want to purge the package data files that are
-
-      downloaded from a Yum repository and held in a cache directory.'
+  - markdown: URL to the directory where the Yum repository's 'repodata' directory
+      lives. Can be an `http://`, `https://` or a `ftp://` URLs. You can specify multiple
+      URLs in one baseurl statement.
 - property: clean_metadata
   ruby_type: true, false
   required: false
   default_value: 'true'
-  new_in: null
+  new_in:
   description_list:
-  - markdown: 'Specifies whether you want to purge all of the packages downloaded
-
-      from a Yum repository and held in a cache directory.'
+  - markdown: Specifies whether you want to purge all of the packages downloaded from
+      a Yum repository and held in a cache directory.
 - property: cost
   ruby_type: String
   required: false
-  default_value: null
-  new_in: null
+  default_value:
+  new_in:
   description_list:
-  - markdown: 'Relative cost of accessing this repository. Useful for weighing one
-
-      repo''s packages as greater/less than any other.'
+  - markdown: Relative cost of accessing this repository. Useful for weighing one
```

Comment:
Do you mean cost in terms of $ or by some other metric (time, etc.)?

Reply:
Computery costs? 🤷

Reply:
FWIW this is pretty much word for word copied from the [official description](https://linux.die.net/man/5/yum.conf): 

> cost relative cost of accessing this repository. Useful for weighing one repo's packages as greater/less than any other. defaults to 1000 

[Ansible uses the same description](https://docs.ansible.com/ansible/latest/modules/yum_repository_module.html).

Seems like an awkward way of describing priority?

Reply:
At least we're not alone in our vagueness.